### ISSUE TITLE
Effect of the number of learnable token and their dimension

### ISSUE THREAD

**Issue by bqm1111** [ShaohuaDong2021/DPLNet#9]
Do you use the same dimension and number of learnable tokens in every stage? Did conduct any ablation study on how the variation of these hyperparameters affects overall performance?

**Comment by ShaohuaDong2021** [comment]
Yes, we used the same dimension and number of learnable tokens in every stage, we conducted the ablation studies for dimension and number of learnable tokens, as shown in Table 6. We did not use the different dimension or number of learnable tokens for different stages.

**Comment by bqm1111** [comment]
I dont see you mention about learning rate scheduler. Did you fix the learning rate for the training process?

**Comment by ShaohuaDong2021** [comment]
Yes, we fixed the learning rate and we initialized the learnable token randomly.
